src/Microsoft.AspNet.Mvc.ViewFeatures/Controller.cs

```diff
@@ -42,7 +42,7 @@ public IServiceProvider Resolver
         {
             get
             {
-                return ActionContext?.HttpContext?.RequestServices;
+                return ControllerContext?.HttpContext?.RequestServices;
```

Comment:
`ControllerContext` is guaranteed to be non-null. Remove the null safe accessor for these properties.